src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import java.util.*;
```

Comment:
Hi Jaeho!! I feel it would be good to import the specific classes you need from the java util class as it is a good practice to avoid using the wildcard (`*`) :)

Reply:
Thanks for the suggestion!